R/get_retweeted_by.R

```diff
@@ -0,0 +1,49 @@
+#' Get users who has retweeted a tweet
+#' 
+#' This function fetches users who has retweeted a tweet
+#'
+#' @param x string containing one tweet id or a vector of tweet ids
+#' @param bearer_token string, bearer token
+#' @param file string, name of the resulting RDS file
+#' @param data_path string, if supplied, fetched data can be saved to the designated path as jsons
+#' @param export_query If `TRUE`, queries are exported to data_path
+#' @param bind_tweets If `TRUE`, tweets captured are bound into a data.frame for assignment
+#' @param verbose If `FALSE`, query progress messages are suppressed
+#'
+#' @return a data frame
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' tweets <- c("1392887366507970561","1409931481552543749")
+#' get_retweeted_by(tweets, bearer_token = get_bearer())
+#' }
+get_retweeted_by <-
+  function(x,
+           bearer_token = get_bearer(),
+           file = NULL,
```

Comment:
If one specifies `file` here, the RDS file will be overwritten in every step, right?